### DISCUSSION TITLE
Null value not displaying in response example

### DISCUSSION THREAD

**Discussion by rusih100** [fastapi/fastapi#13103] [Questions] [answered]
### Example Code

```python
from fastapi import FastAPI, Response
from pydantic import BaseModel

app = FastAPI()


class UserSchema(BaseModel):
    name: str


class ResponseSchema(BaseModel):
    user: UserSchema | None


DATA = {
    1: UserSchema(name="Mike")
}


@app.get(
    "/{key}",
    responses={
        404: {
            "model": ResponseSchema,
            "content": {
                "application/json": {
                    "example": {
                        "user": None
                    }
                }
            }
        }
    }
)
async def root(key: int, response: Response) -> ResponseSchema:
    user = DATA.get(key)
    response.status_code = 200 if user is None else 404
    return ResponseSchema(
        user=user
    )
```

### Description

Either I'm doing something wrong, or something isn't working.
I wanted to add an example response in "Responses" that would display JSON with an empty attribute, but it doesn't work.

### Operating System

Linux

### FastAPI Version

0.115.6

### Pydantic Version

2.10.4

### Python Version

3.12

### Additional Context

![image](https://github.com/user-attachments/assets/571d62ff-3dab-45a2-ace3-9db3a3fe96d2)

**Comment by JavierSanchezCastro** [comment, marked as answer]
Hello!

That behaviour has been going on for some time, have a look at this discussion https://github.com/fastapi/fastapi/discussions/8401 and then try this PR https://github.com/fastapi/fastapi/pull/5693

**Reply by rusih100** [reply]
Thanks!
